Sources/Valet/Internal/SecItem.swift

```diff
@@ -21,51 +21,17 @@
 import Foundation
 
 
-internal func execute<ReturnType>(in lock: NSLock, block: () -> ReturnType) -> ReturnType {
+internal func execute<ReturnType>(in lock: NSLock, block: () throws -> ReturnType) rethrows -> ReturnType {
     lock.lock()
     defer {
         lock.unlock()
     }
-    return block()
+    return try block()
 }
 
 
 internal final class SecItem {
     
-    // MARK: Internal Enum
-    
-    internal enum DataResult<SuccessType> {
```

Comment:
Excited that we can get rid of these types.

Reply:
(Note we could still get rid of these even if we didn’t move to a `throws` API, since Swift 5 has a built-in `Result` type)